internal/batches/executor/changeset_specs.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/sourcegraph/src-cli/internal/batches"
 )
 
+var errOptionalPublishedUnsupported = errors.New(`This Sourcegraph version requires the "published" field to be specified in the batch spec; upgrade to be able to omit the published field and control publication from the UI.`)
```

Comment:
Should we say that they need to upgrade to at least 3.29.0, or "upgrade to the latest", or leave it as it is? (I don't have a strong feeling one way or the other, but wanted to bring it up.)

Reply:
I updated this to include 3.30.0 as the version. (If I'm wrong, we can change that tomorrow.)